dist/output.css

```diff
@@ -1258,8 +1258,8 @@ select {
   max-width: 56rem;
 }
 
-.max-w-\[150px\] {
-  max-width: 150px;
+.max-w-\[145px\] {
+  max-width: 145px;
```

Comment:
how do we know if this is sufficient?

Reply:
looks fine even on small 320px width mobile
<img width="480" alt="CleanShot 2023-02-28 at 12 28 56@2x" src="https://user-images.githubusercontent.com/5802047/221772216-d756bf4a-ccfb-43cf-bd96-dfb65ed0c333.png">

Reply:
Will `grow` + remove flex-wrap on `<nav>` help here so that the title always fills up whatever horizontal size is remaining on the first row?

Reply:
like in the all rooms page? to align title from left instead of center?

Reply:
No it should still be aligned center but that can be done with a text align center. This is just so that the header is wider for wider devices and narrower for narrower devices but the width is always enough to fit on the same row

Reply:
just pushed something

Reply:
fuck pushed to a wrong branch hold on

Reply:
merged changes here